lib/nerves_pack/wifi_wizard_button.ex

```diff
@@ -66,33 +69,38 @@ defmodule NervesPack.WiFiWizardButton do
     if Code.ensure_compiled?(GPIO) do
       GenServer.start_link(__MODULE__, opts, name: __MODULE__)
     else
-      Logger.warn("""
-      [NervesPack] - Skipping WiFiWizardButton: add {:circuits_gpio, "~> 0.4"} to your dependencies to use
-      """)
+      _ =
+        Logger.warn("""
+        [NervesPack] - Skipping WiFiWizardButton: add {:circuits_gpio, "~> 0.4"} to your dependencies to use
+        """)
 
       :ignore
     end
   end
 
   @impl true
   def init(opts) do
-    gpio_pin = opts[:pin] || Application.get_env(:nerves_pack, :wifi_wizard_button_pin, 26)
+    gpio_pin =
+      opts[:pin] || Application.get_env(:nerves_pack, :wifi_wizard_button_pin, @default_pin)
+
     {:ok, gpio} = GPIO.open(gpio_pin, :input)
     :ok = GPIO.set_interrupts(gpio, :both)
 
-    hold = opts[:hold] || Application.get_env(:nerves_pack, :wifi_wizard_button_hold, 5_000)
+    hold =
+      opts[:hold] || Application.get_env(:nerves_pack, :wifi_wizard_button_hold, @default_hold)
 
-    timeout = validate_timeout(hold)
+    hold = validate_timeout(hold)
```

Comment:
Probably should rename this function to `validate_hold` here as well to match these changes

Reply:
The process would be to make that change in another PR. I'm +1 on making that change, though.